package.json

```diff
@@ -7,10 +7,27 @@
     "build": "webpack --config webpack.config.js",
     "build-watch": "webpack --watch --config webpack.config.js",
     "dev-server": "webpack-dev-server --config webpack.config.js",
+    "test": "./node_modules/.bin/jest",
```

Comment:
This should also run `tslint` so we don't miss out on linting errors.

Reply:
@mbebenita  Done :)